docs/installation/opni_agent/index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Opni Agent Installation
+slug: /installation/opni_agent
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+There are two ways to install Opni agent. We recommend installation through the Rancher UI.
+
+<Tabs>
+<TabItem value="rancher" label="Installation using Rancher UI" default>
+Opni provides GitHub charts repository that can be used with the Rancher UI.  To add the charts go to Apps -> Repositories in the Rancher UI.  Name the repository and select the 'Git repository containing Helm chart or cluster template definitions' option.
+
+Enter the following git url:
+```
+https://github.com/rancher/opni.git
+```
+
+And the following branch:
+```
+charts-repo
+```
+
+Once the repo has updated you should be able to find Opni in the list of charts.
+![Opni Charts](/img/opnicharts.png)
+
+ To install the Opni Agent fill in the three required fields:
+ ![Opni Agent](/img/opniagentchart.png)
+
+These can be obtained from the Opni Management UI.  Under cluster select Add Cluster and in the following screen expand the Manual Install Information section
+ ![Add cluster](/img/addcluster.png)
+
+ These fields map directly to the fields in the chart install UI.
+
+</TabItem>
+<TabItem value="helm" label="Installation using Helm">
+
+Agents can be installed on any Kubernetes cluster using the Agent Helm chart.
+
+## Using the Opni Dashboard
+
+Follow these steps to install an agent into a new cluster using the Opni dashboard:
+
+1. Navigate to the Opni dashboard
+
+  To access the dashboard, you can port-forward:
+
+  ```bash
+  kubectl -n opni port-forward svc/opni-admin-dashboard web:web
+  ```
+  Then navigate to [http://localhost:12080](http://localhost:12080).
+
+
+2. Select "Clusters" from the left sidebar, then click "Add Cluster".
+
+  <div className="image-border">
+    <img
+      src={require('/img/installation/add-cluster.png').default} 
+      alt="Add Cluster"
+    />
+  </div>
+
+  <br />
+  <br />
+
+  :::note
+
+  On a new installation, you will see the Opni cluster itself listed here.
+
+  :::
+
+3. On the next screen, adjust the install command settings as needed depending on your environment, then copy the command shown here.
+
+  <div className="image-border">
+    <img
+      src={require('/img/installation/install-command.png').default} 
+      alt="Install Command"
+    />
+  </div>
+
+  
+  <br />
+  <br />
+
+  :::tip
+
+  If you don't have Prometheus Operator installed the cluster you want to add, check the "Install Prometheus Operator" box. If you already have Prometheus Operator installed, it must be a relatively recent version.
```

Comment:
```suggestion
  If you don't have Prometheus Operator installed on the cluster you want to add, check the "Install Prometheus Operator" box. If you already have Prometheus Operator installed, it must be a relatively recent version.
```

Reply:
`relatively recent` is a bit ambiguous here without a point of reference. Is there a known minimum or compatible version that can be added instead?